sass/journey_section.sass

```diff
@@ -0,0 +1,84 @@
+.journey
+  +mainSection
+
+  &__header
+    text-align: center
+
+  &__services
+    display: grid
+    grid-template-columns: 1fr 1fr
+
+    .service
+      &__element
+        align-self: center
+        justify-self: center
+        margin: $m-base
+
+      &__title
+        font-size: $font-size-xs
+        font-weight: normal
+        margin: $m-base auto
+        text-align: center
+
+.icon__container
+  align-items: center
+  background-color: $white
+  border-radius: 50%
+  border: 0 solid $white
+  box-shadow: $shadow $gray
+  color: $secondary
+  display: flex
+  height: $icon-lg
+  justify-content: center
+  margin: 0 auto
+  position: relative
+  width: $icon-lg
+
+  .icon
+    fill: $secondary
+
+    path
+      fill: $secondary
+
+  &:hover
+    background-color: $secondary
+    box-shadow: $shadow-lg $secondary
+    color: $white
+
+    .icon
+      fill: $white
+
+      path
+        fill: $white
+
+    &:after
+      border-radius: 50%
+      border: $border-base solid $white
+      content: ''
+      display: block
+      height: $icon-lg*1.5
+      position: absolute
+      width: $icon-lg*1.5
+
+@media (min-width: $min-width)
+  .journey
+
+    &__header
+      margin: auto
+      text-align: center
+      width: 60%
+
+    &__services
+      grid-template-columns: 1fr 1fr 1fr
```

Comment:
grid is not in the scope of this level, but you can also write it as `repeat(3, 1fr)`